.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+# HMS Networks Solution Center
+# Artifact Deploy Action for this project
+# Version: 1.0
+#
+name: Deploy Assets S3
```

Comment:
```suggestion
name: Release (Deploy to S3)
```

Reply:
I think this script should be renamed to better match the other release actions:

![image](https://user-images.githubusercontent.com/51409174/232539634-4215d57f-0bb3-4a62-b6da-22340cd947ff.png)